scene/main/node.cpp

```diff
@@ -1899,6 +1899,56 @@ Node *Node::get_deepest_editable_node(Node *p_start_node) const {
 	return node;
 }
 
+#ifdef TOOLS_ENABLED
+void Node::set_property_pinned(const String &p_property, bool p_pinned) {
+	bool current_pinned = false;
+	bool has_pinned = has_meta(SNAME("_pinned_properties_"));
```

Comment:
As this is an editor-related meta, I would prefix this with "_edit", as we already have "_edit_group", "_edit_lock", "_edit_vertical_guides", etc...
This should be very helpful if we decide to get rid of editor-created metadata, or move them to another property.

Reply:
Done! (Also in the 3.x version.)

Reply:
By the way, I've removed the `SNAME()` wrapping for consistency (I've realized other parts of the code don't use if for meta keys). My plan is at some point to check if there's a possibility to improve that, so that maybe meta lookups can be faster if completely based on `StringName`.

Reply:
Done.